messaging.rst

```diff
@@ -31,22 +31,18 @@ Fields
 +==========================+============+================================================================================+
 |  nonce                   | uint256    | Strictly monotonic value used to order transfers. The nonce starts at 1        |
 +--------------------------+------------+--------------------------------------------------------------------------------+
-|  transferred_amount      | uint256    | Total transferred amount in the history of the channel (monotonic value)       |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-|  locked_amount           | uint256    | Current locked amount                                                          |
-+--------------------------+------------+--------------------------------------------------------------------------------+
```

Comment:
I shouldn't have removed these two fields above.  I was again confused about the onchain/offchain formats.  Will fix.

Reply:
I've put those fields in again.